src/SimpleThings/EntityAudit/AuditConfiguration.php

```diff
@@ -35,6 +35,17 @@ class AuditConfiguration
     private $currentUsername = '';
     private $revisionIdFieldType = 'integer';
 
+    public function getTableName($metadata)
+    {
+        $tableName = $metadata->getTableName();
+
+        if ($metadata->getSchemaName()) {
```

Comment:
@UFTimmy this isn't working for orm < 2.5. please add a check if the class method exists.